apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -420,47 +451,57 @@ function initializeBlocklyWrapper(blocklyInstance) {
   };
 
   const googleBlocklyMixin = blocklyWrapper.BlockSvg.prototype.mixin;
-  blocklyWrapper.BlockSvg.prototype.mixin = function (
-    mixinObj,
-    opt_disableCheck
-  ) {
+  blocklyWrapper.BlockSvg.prototype.mixin = function (mixinObj) {
     googleBlocklyMixin.call(this, mixinObj, true);
   };
 
-  blocklyWrapper.BlockSvg.prototype.isDisabled = function () {
-    return this.disabled;
+  const extendedBlockSvg = blocklyWrapper.BlockSvg
+    .prototype as ExtendedBlockSvg;
+
+  extendedBlockSvg.isDisabled = function () {
+    return !this.isEnabled();
```

Comment:
Do we even need this at all? I'm pretty sure that the only place we are calling `isDisabled` in a Blockly context is here:

https://github.com/code-dot-org/code-dot-org/blob/65b3753faa7ed11056ef03dc08728552f7282232/apps/src/blockly/addons/cdoGenerator.js#L51

If we updated the above call to use the built-in `isEnabled`, I think we could remove this particular extended function. 

Not sure if you're try trying to avoid diffs in other files here or not. If you think it's cleaner to leave for now and do it as a cleanup that's fine with me.

Reply:
Yeah I wanted to keep this as scoped as possible. I made a [follow-up ticket](https://codedotorg.atlassian.net/browse/CT-343) and will add your suggestions to it!